### PR TITLE
Inaki/dropdown component

### PR DESCRIPTION
**Pull request by inakiaranzadi-optimizely**
Beta version of the Dropdown components for OUI based on Optimizely existing implementation.

@daverau-optimizely

### CODE REVIEW

File: src/components/Dropdown/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+class Dropdown extends React.Component {
+  render() {
+    const {
+      children,
+      handleClick,
+      icon = 'chevron',
+      isDisabled,
```
Comment (jackschlotthauer-optimizely) [inline src/components/Dropdown/index.js]: you might want to default isDisabled to false, isOpen = false (when will a drop down start open?) and width to something like 200px (which is what the storybook uses.

File: src/components/Dropdown/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      isOpen,
+      fullWidth,
+      style,
```
Comment (jackschlotthauer-optimizely) [inline src/components/Dropdown/index.js]: is this a style object? or a parameter for a classname?

Comment (inakiaranzadi-optimizely) [reply src/components/Dropdown/index.js]: good call, I'll add a classname prop good call.

File: src/components/Dropdown/index.js (hunk 1 above)
Comment (inakiaranzadi-optimizely) [reply src/components/Dropdown/index.js]: Good suggestion.

**Pull request merged**